Editor/MenuActions/Editors/NewPolyShape.cs

```diff
@@ -34,6 +34,23 @@ public override bool enabled
 
         public override ActionResult DoAction()
         {
+            var inspWindow = EditorWindow.GetWindow<InspectorWindow>();
+            if (inspWindow != null && inspWindow.isLocked == true)
+            {
+                if (UnityEditor.EditorUtility.DisplayDialog(
+                                    "Inspector Locked",
```

Comment:
User facing text needs to be localized. Use `L10n.Tr` or `EditorGUIUtility.TrTextContent`

Reply:
Thanks! Will do!

Reply:
BTW none of the other DisplayDialog uses the localized version.